app/web/src/features/auth/signup/AccountForm.tsx

```diff
@@ -276,7 +258,21 @@ export default function AccountForm() {
                   HostingStatus.HOSTING_STATUS_MAYBE,
                   HostingStatus.HOSTING_STATUS_CANT_HOST,
                 ]}
-                optionLabelMap={{ "": "", ...hostingStatusLabels }}
+                optionLabelMap={{
+                  "": "",
+                  [HostingStatus.HOSTING_STATUS_CAN_HOST]: t(
+                    "auth:account_form.hosting_status.can_host"
+                  ),
+                  [HostingStatus.HOSTING_STATUS_MAYBE]: t(
+                    "auth:account_form.hosting_status.maybe"
+                  ),
+                  [HostingStatus.HOSTING_STATUS_CANT_HOST]: t(
+                    "auth:account_form.hosting_status.cant_host"
+                  ),
+                  [HostingStatus.HOSTING_STATUS_UNSPECIFIED]:
+                    t("global:ask_me"),
+                  [HostingStatus.HOSTING_STATUS_UNKNOWN]: t("global:ask_me"),
+                }}
```

Comment:
I was wondering about this - will we definitely have to move all label maps like this into components?

Reply:
I think it's the most reliable that way, since it integrates with Suspense and ensures i18next is fully initialised before rendering this. Otherwise, it's possible to use the `t` function before i18next has finished initialising if the user loads this page straight away.

I ran into this problem in tests because of said issue, but in tests at least I managed to work around it by doing a `waitFor` on `isInitialized` until it's true. Not sure how we'd work around that if this object is still statically listed in a constant file though when `t` is not ready to be used.